### ISSUE TITLE
s-push-url attribute doesn't work anymore with 1.1.0 release

### ISSUE THREAD

**Issue by billmn** [putyourlightson/craft-sprig#62]
### Describe the bug
With the 1.1.0 release the `s-push-url` seems that it doesn't work anymore.

Before this release, when I clicked on the example link, the variant id in the url was changed (e.g. http://domain.test/slug?variant=99 -> http://domain.test/slug?variant=32) but this no longer happens.

```twig
<a href="{{ variant.url }}" sprig s-push-url="?variant={{ variant.id }}" s-val:variant="{{ variant.id }}">
    {{ variant.title }}
</a>
```

Using `hx-push-url` it works fine.

### Versions
- Plugin version: 1.1.0
- Craft version: 3.5.15.1

**Comment by bencroker** [comment]
Thanks, fixed in https://github.com/putyourlightson/craft-sprig/commit/1912b3030cafab504c60f8670115766031ab1e81 for the next release.

**Comment by bencroker** [comment]
Released in 1.1.1.
